docs/_includes/projects/try/create/after/index.html

```diff
@@ -9,6 +9,6 @@
   <title>lit-element code sample</title>
 </head>
   <body>
-    <custom-element></custom-element>
+    <my-element></my-element>
```

Comment:
I think we need to educate people to always prefix with a project name like `<mwc-button>` instead of doing things like `<card-viewer>` which will lead to conflicting element names

Reply:
I'd recommend against `foo/bar` since they are abstract concepts and [don't help new developers](https://dev.to/warrend/no-more-foo-bar-and-baz-56j4). Ideally a realistic example and and related name would be used.

Reply:
Thanks both. I'll avoid `foo/bar`. 

I also don't want to give the impression that this docset will tell users how to build a whole app or a suite of elements though, because at the moment that's not in scope.

I'll see what I can come up with for more realistic demo elements and names. (I had thought about trying to make the tutorial element more useful than just plopping out meaningless test strings but my priority is to get the syntax down. TODO..)

Reply:
Simple generic element names bring home the message best and don't confuse, but a one-liner regarding best practice/namespacing at the bottom could be instructive.